fuck

```diff
@@ -1,2 +1,4 @@
 It will be test product
 Second str is become
+
+Fuuuck
```

Comment:
This is not what i expected to see:) 
i expected to see all your new changes in .py class(like all methods etc. in this 'green' fashion - as it was freshly added. 
but - congrats with creating new PR:)